docs/layouts/_default/list.html

```diff
@@ -2,9 +2,21 @@
 <div class="td-content">
 	<h1>{{ .Title }}</h1>
         {{ with .Params.description }}<div class="lead">{{ . | markdownify }}</div>{{ end }}
-	{{ if (and (not .Params.hide_readingtime) (.Site.Params.ui.readingtime.enable)) }}
-            {{ partial "reading-time.html" . }}
-        {{ end }}
+	<header class="article-meta">
```

Comment:
The effort here (and for all of the `layouts` files here in this PR) was to migrate the content from the default hugo templates into ours -- when we created the site initially the templates were customized and the taxonomy pieces were taken out, this adds them back where necessary.  The end result is effectively the default hugo/docsy visualization of taxonomy.